releasenotes/notes/add-metadata-grouper-21ec05fd4a307425.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    We have added a new MetaDataGrouper component that reorders documents by grouping them based on metadata keys. This can be useful pre-processing before feeding them to an LLM.
```

Comment:
```suggestion
    We have added a new MetaFieldGroupingRanker component that reorders documents by grouping them based on metadata keys. This can be useful for pre-processing Documents before feeding them to an LLM.
```